apps/system/lockscreen/js/lockscreen_inputpad.js

```diff
@@ -147,7 +147,8 @@
   };
 
   LockScreenInputpad.prototype.updatePassCodeUI =
-  function() {
+  function(keyCode) {
```

Comment:
I haven't figured out where this keycode is coming from. Where's the code that listens to key events from the keypad?  And are you going to be removing the unnecessary code for displaying and a keypad and handling events on it?